gradle.properties

```diff
@@ -13,6 +13,7 @@ fabric_version=0.4.29+build.290-1.15
 libcd_version=2.1.1+1.15.1
 cotton_resources_version=1.4.1+1.14.4
 shulkerboxtooltip_version=1.4.5+1.15
+quickshulker_version=-SNAPSHOT
```

Comment:
Maybe try the commit for your version? Jitpack is weird.

Reply:
I tried multiple versions including using mavenLocal rather than jitpack. I’m assuming it’s an issue on quick shulkers end.

Reply:
Maybe try to setup something on bintray? I would like it to be possible to just be able to straight import the project without having to manually include your jar or make someone download it.